From 626ee8f3fb2f63029bfe80949f2736974480b069 Mon Sep 17 00:00:00 2001
From: Kazunori Kobayashi <kkobayas@igel.co.jp>
Date: Thu, 8 Nov 2012 14:20:18 +0900
Subject: [PATCH 94/97] dfb-example/dfb-video-example: remove clearing a
 surface before the pipeline running

It is not necessary because a external surface will be cleared
at the initialization of dfbvideosink.
---
 ext/directfb/dfb-example.c       |    4 ----
 ext/directfb/dfb-video-example.c |    4 ----
 2 files changed, 8 deletions(-)

diff --git a/ext/directfb/dfb-example.c b/ext/directfb/dfb-example.c
index 5214218..2f14094 100644
--- a/ext/directfb/dfb-example.c
+++ b/ext/directfb/dfb-example.c
@@ -206,10 +206,6 @@ main (int argc, char *argv[])
       g_error ("Couldn't link src and sink");
   }
 
-  primary->Clear (primary, 0x00, 0x00, 0x00, 0xFF);
-  primary->Flip (primary, NULL, DSFLIP_NONE);
-  primary->Clear (primary, 0x00, 0x00, 0x00, 0xFF);
-
   /* Let's play ! */
   gst_element_set_state (pipeline, GST_STATE_PLAYING);
 
diff --git a/ext/directfb/dfb-video-example.c b/ext/directfb/dfb-video-example.c
index 99e7644..424dc1f 100644
--- a/ext/directfb/dfb-video-example.c
+++ b/ext/directfb/dfb-video-example.c
@@ -599,10 +599,6 @@ main (int argc, char *argv[])
 
   g_signal_connect (demuxer, "pad-added", G_CALLBACK (on_pad_added), sink);
 
-  primary->Clear (primary, 0x00, 0x00, 0x00, 0xFF);
-  primary->Flip (primary, NULL, DSFLIP_NONE);
-  primary->Clear (primary, 0x00, 0x00, 0x00, 0xFF);
-
   /* set signal handler */
   action.sa_handler = play_handler;
   sigaction (SIGQUIT, &action, NULL);
-- 
1.7.10.4

